src/Symfony/Component/Yaml/composer.json

```diff
@@ -24,7 +24,7 @@
         "symfony/console": "^4.4|^5.0"
     },
     "conflict": {
-        "symfony/console": "<4.4"
+        "symfony/console": "<5.3"
```

Comment:
Is this bump really necessary? I think it would be okay if the GitHub formatting is simply not available with Console 4.4.

Reply:
You're right, I prefer this as well. I'll change it, thanks.